src/DevShop/Component/DevShopControlTemplate/scripts/composer/ScriptHandler.php

```diff
@@ -14,6 +14,24 @@
 
 class ScriptHandler {
 
+  public static function checkDevmasterPackage(Event $event) {
+    $fs = new Filesystem();
+    $io = $event->getIO();
+    $drupalFinder = new DrupalFinder();
+    $drupalFinder->locateRoot(getcwd());
+    $drupalRoot = $drupalFinder->getDrupalRoot();
+
+    $devmaster_path = $drupalRoot . '/profiles/devmaster';
+    $devmaster_info_path = $devmaster_path  . '/devmaster.info';
+    if (!$fs->exists($devmaster_info_path)) {
+      $io->writeError('<error>Devmaster profile info file was not found at ' . $devmaster_info_path . '</error>');
```

Comment:
Ah this is the error message I saw in the test that failed. 
`if (!$fs->exists($devmaster_info_path))` returns TRUE in the automatic test.

Reply:
Here's the current string that should be the path to the .info file.
`/usr/share/devshop/src/DevShop/Component/DevShopControlTemplate/web/profiles/devmaster/devmaster.info`

Reply:
Removing the preserve path fixes it from installing directly.

Now it's failing when using the custom composer "repository" that installs via symlink.

Whipping up code to detect the symlink and try to read the full path.

The path it's looking for is in https://github.com/opendevshop/devshop/pull/632/checks?check_run_id=1449549897#step:4:648

/usr/share/devshop/src/DevShop/Component/DevShopControlTemplate/web/profiles/devmaster/devmaster.info